bin/tfidf_search.py

```diff
@@ -0,0 +1,97 @@
+import argparse
+import os
+import sqlite3
+import pandas as pd
+from sklearn.metrics.pairwise import linear_kernel
+from sklearn.feature_extraction.text import TfidfVectorizer
```

Comment:
Not sure if you can do this in Python, but in Ruby you can rescue on an import and print a message, like: `Please run pip3 install scikit-learn pandas` before using this script.

Reply:
Will look into this... It definitely would be helpful!